x-pack/solutions/security/plugins/security_solution/public/entity_analytics/components/threat_hunting_entity_risk_levels.tsx

```diff
@@ -0,0 +1,305 @@
+    { defaultMessage: 'Number of entities' }
+  ),
+  dataType: 'number',
+  width: '35%',
+};
+
+const RISK_LEVEL_CONFIG: Array<{
+  severity: RiskSeverity;
+  label: string;
+  range: string;
+}> = [
+  {
+    severity: RiskSeverity.Critical,
+    label: i18n.translate(
+      'xpack.securitySolution.entityAnalytics.threatHunting.riskLevels.critical',
+      {
+        defaultMessage: 'Critical',
+      }
+    ),
+    range: '> 90',
+  },
+  {
+    severity: RiskSeverity.High,
+    label: i18n.translate('xpack.securitySolution.entityAnalytics.threatHunting.riskLevels.high', {
+      defaultMessage: 'High',
+    }),
+    range: '70 - 90',
+  },
+  {
+    severity: RiskSeverity.Moderate,
+    label: i18n.translate(
+      'xpack.securitySolution.entityAnalytics.threatHunting.riskLevels.medium',
+      {
+        defaultMessage: 'Medium',
+      }
+    ),
+    range: '40 - 70',
+  },
+  {
+    severity: RiskSeverity.Low,
+    label: i18n.translate('xpack.securitySolution.entityAnalytics.threatHunting.riskLevels.low', {
+      defaultMessage: 'Low',
+    }),
+    range: '20 - 40',
+  },
+  {
+    severity: RiskSeverity.Unknown,
+    label: i18n.translate(
+      'xpack.securitySolution.entityAnalytics.threatHunting.riskLevels.unknown',
+      {
+        defaultMessage: 'Unknown',
+      }
+    ),
+    range: '< 20',
+  },
+];
+
+const createEmptySeverityCount = (): SeverityCount => ({
+  [RiskSeverity.Unknown]: 0,
+  [RiskSeverity.Low]: 0,
+  [RiskSeverity.Moderate]: 0,
+  [RiskSeverity.High]: 0,
+  [RiskSeverity.Critical]: 0,
+});
+
+const RiskLevelsTable = ({ items, loading }: { items: RiskLevelDatum[]; loading: boolean }) => (
+  <EuiBasicTable
+    columns={[RISK_LEVEL_COLUMN, RANGE_COLUMN, COUNT_COLUMN]}
+    items={items}
+    loading={loading}
+    responsiveBreakpoint={false}
+  />
+);
+
+const donutHeight = 180;
+
+const RiskLevelsPie = ({
+  data,
+  fillColor,
+  totalCount,
+}: {
+  data: DonutChartProps['data'];
+  fillColor: DonutChartProps['fillColor'];
+  totalCount: number;
+}) => (
+  <DonutChart
+    data={data}
+    fillColor={fillColor}
+    height={donutHeight}
+    title={<ChartLabel count={totalCount} />}
+    label={i18n.translate(
+      'xpack.securitySolution.entityAnalytics.threatHunting.riskLevels.totalEntities',
+      { defaultMessage: 'entities' }
+    )}
+    totalCount={totalCount}
+  />
+);
+
+export const ThreatHuntingEntityRiskLevels: React.FC = () => {
+  const { from, to } = useGlobalTime();
+  const { filterQuery } = useGlobalFilterQuery();
+  const timerange = useMemo(
+    () => ({
+      from,
+      to,
+    }),
+    [from, to]
+  );
+
+  const entityTypes = useEntityAnalyticsTypes();
+  const includeHost = entityTypes.includes(EntityType.host);
+  const includeUser = entityTypes.includes(EntityType.user);
+  const includeService = entityTypes.includes(EntityType.service);
+
+  const toggleId = 'threat-hunting-entity-risk-levels';
```

Comment:
This is supposed to be the query ID. So it doesn't run the query when toggle is off